### PR TITLE
Clear Keybindings functionality

### PR DESCRIPTION
**Pull request by raghuvanshy**
Hello all,
The pr adds the ability to reset keybindings so as to start afresh. this is done by simply `ui.Keybindings = make([]*keybindings, 0)`
Your feedback is more than welcome

### CODE REVIEW

**Comment by marcusolsson** [comment]
Thanks for wanting to contribute!

Just spitballing here but to me the word "Reset" implies returning to a semantically default state. This is sort of the case here but my gut feeling would tell me that "Reset" could be something more than just removing all keybindings. For example, if there were standard keybindings like Esc for quitting, that would persist even after removing them, then Reset could be appropriate. How about "ClearKeybindings" maybe?

**Comment by raghuvanshy** [comment]
Hmm, I take your point. For example for my project I may use ResetKeybindings() to first clear all keybindings and then set the exit mechanism.

File: ui_tcell.go [hunk 1]
```diff
@@ -64,6 +64,12 @@ func (ui *tcellUI) SetKeybinding(seq string, fn func()) {
 	})
 }
 
+// ClearKeygindings reinitializes the keybindings so as to revert to the
```
Comment (marcusolsson) [inline ui_tcell.go]: Typo in documentation.

**Comment by marcusolsson** [comment]
Thank you! 👍

**Pull request merged**